src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,71 @@
+package edu.kis.vh.nursery;
+import storage.IntLinkedList;
+
+public class DefaultCountingOutRhymer implements Generate {
+	
+	private IntLinkedList intLinkList =  new IntLinkedList();
+	
+	public DefaultCountingOutRhymer(IntLinkedList intLinkList) {

```

Comment:
Only IntLinkedList can be used here. The goal was to enable using IntArrayStack and IntLinkedList.